components/automate-deployment/pkg/majorupgradechecklist/v3.go

```diff
@@ -253,19 +249,59 @@ func backupCheck() Checklist {
 	}
 }
 
-func diskSpaceCheck() Checklist {
+var SpaceAvailable bool
```

Comment:
Are we using this variable value in other files? If not then make its scope local, i.e. declare it inside `diskSpaceCheck` function.